suites/nautilus/rbd/tier-1_rbd.yaml

```diff
@@ -86,7 +86,7 @@ tests:
       polarion-id: CEPH-83574528
    - test:
        config:
-         branch: master
+         branch: nautilus
```

Comment:
Could you include the logs for this suite (main and nautilus)?

Reply:
There seemed to be some pre-existing issue with nautilus upstream script which I did not know, sorry.
`main` is working fine, made changes accordingly 
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1TF8DJ/

Reply:
Marking as resolved for now as file no longer exists in the PR.